nomad-aws/security_groups.tf

```diff
@@ -8,14 +8,14 @@ resource "aws_security_group" "nomad_sg" {
     from_port   = 64535
     to_port     = 65535
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
+    cidr_blocks = ["0.0.0.0/0"] #tfsec:ignore:aws-ec2-no-public-ingress-sgr
```

Comment:
Temporarily ignore these failing `tfsec` checks introduced in PR #183 until we can properly action them as they're failing CI and making detecting other issues more difficult.

Reply:
Pretty sure we actually do want 0.0.0.0 here so we can SSH into jobs